src/components/WebcamCapture/WebcamCapture.js

```diff
@@ -115,37 +116,40 @@ class WebcamCapture extends Component {
       return null;
     }
 
-    if (this.state.cameraConnected) {
-      return (
-        <div className="webcam-container">
-          <Webcam
-            audio={false}
-            width="100%"
-            height="100%"
-            ref={this.webcam}
-            screenshotFormat="image/jpeg"
-            className="webcam-capture--video"
-            screenshotWidth={this.props.imgSize * (4 / 3)}
+    return (
+      <div className="webcam-container">
+        <ViewFinder ref={this.viewFinder} />
+        {this.state.fakeWebcam ? (
+          <input
+            id="fileUpload"
+            type="file"
+            accept="image/*"
+            ref={this.fileInput}
           />
-          <ViewFinder ref={this.viewFinder} />
-          {this.state.fakeWebcam && (
-            <input
-              id="fileUpload"
-              type="file"
-              accept="image/*"
-              ref={this.fileInput}
-            />
-          )}
-        </div>
-      );
-    }
-    return <Notification message="failed to load video feed" isError={true} />;
+        ) : (
+          this.state.cameraConnected && (
+            <div className="webcam-container">
```

Comment:
Have we accidentally re-intorduced this line somewhere? There shouldn't be a `.webcam-container` inside a `.webcam-container`